TwitchPlaysAssembly/Src/ComponentSolvers/Modded/Misc/AbstractSequencesComponentSolver.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+
+public class AbstractSequencesComponentSolver : ReflectionComponentSolver
+{
+	public AbstractSequencesComponentSolver(TwitchModule module) :
+		base(module, "abstractSequencesScript", "!{0} press <p1> (p2)... [Presses the numbered button(s) in the specified position(s)] | !{0} submit [Presses the submit button] | Valid positions are 1-16 in reading order")
+	{
+	}
+
+	public override IEnumerator Respond(string[] split, string command)
+	{
+		if (!command.Equals("submit") && !command.StartsWith("press ")) yield break;
```

Comment:
This seems a bit redundant because both conditions are checked in the following if statement.